app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    source_file = None
+    try:
+        _, source_file, destination_path = command.split()
+
+        if any(char in source_file + destination_path for char in '<>:"|?*'):
```

Comment:
this is unneccessary - if you put `os.makedirs` and `shutil.move` in try-except clauses an exception will be triggered that you are already handling